PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalInternalClientUnitTest.java

```diff
@@ -1107,40 +1108,12 @@ public void sendRequest_propagatesMalformedJSONResponseErrors() {
         );
 
         PayPalCheckoutRequest payPalRequest = new PayPalCheckoutRequest("1.00", true);
-        sut.sendRequest(context, payPalRequest, payPalInternalClientCallback);
+        sut.sendRequest(context, payPalRequest, configuration, payPalInternalClientCallback);
 
         verify(payPalInternalClientCallback).onResult((PayPalPaymentAuthRequestParams) isNull(),
             any(JSONException.class));
     }
 
-    @Test
-    public void sendRequest_onConfigurationFailure_forwardsError() {
```

Comment:
We don't need this test anymore since we aren't fetching a config in the internal client now